### DISCUSSION TITLE
Redis 500ms timeout, any reason for it?

### DISCUSSION THREAD

**Discussion by faridsaud** [directus/directus#10660] [Archive]
Inside the [cache.ts](https://github.com/directus/directus/blob/main/api/src/cache.ts#L58) file, we have the `commandTimeout` set for Redis at 500ms, any specific reason for it?

I was getting some timeout errors with 500ms, removing it seems to do the trick on my setup.

**Comment by rijkvanzanten** [comment]
Cause we don't expect a single command to take longer than 500ms. If it did, it often indicates something is going wrong. What problem were you running into? What commands would take longer than 500ms?

**Comment by rijkvanzanten** [comment]
@faridsaud I tweaked this in https://github.com/directus/directus/pull/10773. You can now leave it omitted (for no timeout), or configure it through `CACHE_REDIS_COMMAND_TIMEOUT` 👍🏻
